fastcrypto-cli/src/vdf.rs

```diff
@@ -157,33 +158,46 @@ fn execute(cmd: Command) -> Result<String, Error> {
         Command::Verify(arguments) => {
             let discriminant_bytes = hex::decode(arguments.discriminant)
                 .map_err(|_| Error::new(ErrorKind::InvalidInput, "Invalid discriminant."))?;
-            let discriminant = Discriminant::try_from_be_bytes(&discriminant_bytes)
+            let discriminant = bcs::from_bytes::<Discriminant>(&discriminant_bytes)
                 .map_err(|_| Error::new(ErrorKind::InvalidInput, "Invalid discriminant."))?;
 
-            let input = QuadraticForm::from_bytes(
-                &hex::decode(arguments.input).map_err(|_| {
+            let input =
+                bcs::from_bytes::<QuadraticForm>(&hex::decode(arguments.input).map_err(|_| {
                     Error::new(ErrorKind::InvalidInput, "Invalid output hex string.")
-                })?,
-                &discriminant,
-            )
-            .map_err(|_| Error::new(ErrorKind::InvalidInput, "Invalid output."))?;
+                })?)
+                .map_err(|_| Error::new(ErrorKind::InvalidInput, "Invalid output."))?;
```

Comment:
InvalidInput Vs Invalid output?

Reply:
Yea, that's unclear. It's the output of the VRF. I'll fix the error message.